resources/dicts/thoughts/dead/starclan/general.json

```
@@ -165,7 +188,8 @@
             "Is excited that {PRONOUN/m_c/subject} {VERB/m_c/are/is} able to help c_n by sharing a new prophecy",
             "Whispers useful suggestions in r_c's ears",
             "Sends a vision of an undiscovered herb patch to r_c",
-            "Lets {PRONOUN/m_c/poss} scent linger near r_c, if only for a moment"
+            "Lets {PRONOUN/m_c/poss} scent linger near r_c, if only for a moment",
+            "Tells r_c where to find an herb c_n is low on"
```

```suggestion
            "Leads r_c to a patch of an herb c_n is low on"
```
i think it should be more subtle than the original, but i'm not totally confident in the phrasing of this edit. could be better?